django/conf/locale/ms/formats.py

```diff
@@ -0,0 +1,36 @@
+# This file is distributed under the same license as the Django package.
+#
+# The *_FORMAT strings use the Django date format syntax,
+# see https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date
+DATE_FORMAT = 'j M Y'                   # '25 Oct 2006'
+TIME_FORMAT = 'P'                       # '2:30 p.m.'
+DATETIME_FORMAT = 'j M Y, P'            # '25 Oct 2006, 2:30 p.m.'
+YEAR_MONTH_FORMAT = 'F Y'               # 'October 2006'
+MONTH_DAY_FORMAT = 'j F'                # '25 October'
+SHORT_DATE_FORMAT = 'd/m/Y'             # '25/10/2006'
+SHORT_DATETIME_FORMAT = 'd/m/Y P'       # '25/10/2006 2:30 p.m.'
+FIRST_DAY_OF_WEEK = 1                   # Monday
+
+# The *_INPUT_FORMATS strings use the Python strftime format syntax,
+# see https://docs.python.org/library/datetime.html#strftime-strptime-behavior
+DATE_INPUT_FORMATS = [
+    '%d/%m/%Y', '%d/%m/%y',             # '25/10/2006', '25/10/06'
+    # '%b %d %Y', '%b %d, %Y',          # 'Oct 25 2006', 'Oct 25, 2006'
+    # '%d %b %Y', '%d %b, %Y',          # '25 Oct 2006', '25 Oct, 2006'
+    # '%B %d %Y', '%B %d, %Y',          # 'October 25 2006', 'October 25, 2006'
+    # '%d %B %Y', '%d %B, %Y',          # '25 October 2006', '25 October, 2006'
```

Comment:
We can chop commented formats.

Reply:
Chopped off unused formats and retained only suitable formats.